src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    protected String task_info;
+    protected boolean task_completion;
+
+    public Task(String task_info) {
+        this.task_info = task_info;
+        this.task_completion = false;
+    }
+
+    public void completeTask() {
+        this.task_completion = true;
+    }
+
+    public boolean get_task_completion() {
```

Comment:
I think based on the coding standard, the function name should be getTaskCompletion.